x-pack/plugins/apm/server/index.ts

```diff
@@ -29,7 +29,7 @@ export function mergeConfigs(apmOssConfig: APMOSSConfig, apmConfig: APMXPackConf
     'apm_oss.sourcemapIndices': apmOssConfig.sourcemapIndices,
     'apm_oss.onboardingIndices': apmOssConfig.onboardingIndices,
     'apm_oss.indexPattern': apmOssConfig.indexPattern,
-    'xpack.apm.servicemapEnabled': apmConfig.servicemapEnabled,
+    'xpack.apm.serviceMapEnabled': apmConfig.serviceMapEnabled,
```

Comment:
Any way to avoid this in the future? I'm not saying we should use constants for config keys... but should we? :p

Reply:
the issue is that it was `xpack.apm.serviceMapEnabled` in the LP config, and after I created an NP config, I typo'ed this specific key. I'm not so sure what could have avoided it, maybe a unit test that bootstraps the plugin and checks the value emitted by the contract's `config$` observable

Reply:
but even then.. probably the unit test would have the typo as well 😅